### DISCUSSION TITLE
How to refresh materialized view if source table was changed

### DISCUSSION THREAD

**Discussion by petrixs** [ClickHouse/ClickHouse#39997] [Q&A] [answered]
Every night we recalculate amount field in source table for last 24h( ALTER TABLE transactions UPDATE amount = round(amount/2, 8) IN PARTITION '20210908' where amount>0; ). We have materialized view (summing merge tree) which aggregate and sum data from milliseconds  to minutes granularity. Does clickhouse support re-aggregation data after update of source table?

**Comment by den-crane** [comment, marked as answer]
>  Does clickhouse support re-aggregation data after update of source table?

No, it does not. 
The only way is to drop MatView and create it again.

**Reply by petrixs** [reply]
Thank you
